cli/subcommands/src/cargo_hax.rs

```diff
@@ -199,7 +199,7 @@ fn run_engine(
 
         let stdout = std::io::BufReader::new(engine_subprocess.stdout.take().unwrap());
         for msg in stdout.json_lines() {
-            let msg = msg.unwrap();
+            let msg = msg.expect("hax engine sent an invalid json value");
```

Comment:
Can you add something like: "This might be caused by debug messages on stdout, which is reserved for JSON communication with `cargo-hax`"?

Reply:
Added now.